modelling/src/test/java/org/axonframework/modelling/command/inspection/AnnotatedAggregateMetaModelFactoryTest.java

```diff
@@ -302,6 +370,33 @@ void typedAggregateIdentifier() {
                 () -> AnnotatedAggregateMetaModelFactory.inspectAggregate(TypedIdentifierAggregate.class));
     }
 
+    @Test
+    void testGetterTypedAggregateIdentifier() {
+        assertThrows(
+                AxonConfigurationException.class,
+                () -> AnnotatedAggregateMetaModelFactory.inspectAggregate(TypedIdentifierAggregate.class));
```

Comment:
Shouldn't `GetterTypedIdentifierAggregate` be used for this test instead of `TypedIdentifierAggregate`?